android/src/main/java/com/magicleap/magicscript/scene/nodes/button/UiButtonNode.kt

```diff
@@ -133,9 +131,9 @@ open class UiButtonNode(
             onPressedChangeListener = { pressed ->
                 val z =
                     if (pressed) {
-                        initialLocalPosition.z + ANIMATED_Z
+                        contentNode.localPosition.z + ANIMATED_Z
                     } else {
-                        initialLocalPosition.z
+                        contentNode.localPosition.z
                     }
                 localPosition = Vector3(localPosition.x, localPosition.y, z)
```

Comment:
Should be `contentNode.localPosition = Vector3(localPosition.x, localPosition.y, z)`